### PR TITLE
Ps 175 add comments to posts

### PR DESCRIPTION
**Pull request by Tibor22**
Users can create posts and comment on posts. The first comment is shown by default. Users can click to show all comments, Comment has a smaller font size than post. Comment show username and time.

Also completed #187

### CODE REVIEW

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
In general it looks good. Well done for updating the JSON fixtures for cypress!

Importantly, however, you've reformatted the entire set of files. Please undo the formatting change - you've gone from 2 spaces to 8spaces/2tabs. This will cause a headache for everyone else in the cohort by introducing constant unnecessary changes to the codebase :)

File: src/components/posts/PostsPage.js [hunk 1]
```diff
@@ -1,54 +1,109 @@
...
+		client
+			.post('/post', post)
+			.then((res) => {
+				setPostResponse(res.data);
+				client.get('/posts').then((res) => {
+					setPosts(res.data.data.posts);
+				});
+			})
+			.catch((data) => {
+				console.log(data);
```
Comment (Auenc) [inline src/components/posts/PostsPage.js:32]: Can we change this to be `console.error` so it displays as a red error message in the console

File: src/components/posts/PostsPage.js [hunk 2]
```diff
@@ -1,54 +1,109 @@
...
+							post.postComments.push(res.data.data);
+							return post;
+						} else {
+							return post;
+						}
+					});
+				});
+			})
+			.catch((data) => {
+				console.log(data);
```
Comment (Auenc) [inline src/components/posts/PostsPage.js:62]: Let's also change this to be a `console.error`

**Review by Auenc — CHANGES_REQUESTED** [review]
Two minor changes

**Review by Auenc — APPROVED** [review]
LGTM!

**Review by Hoolio333 — APPROVED** [review]

**Pull request merged**